hushline/settings.py

```diff
@@ -683,4 +681,26 @@ def delete_account() -> Response | str:
         flash("User not found. Please log in again.")
         return redirect(url_for("login"))
 
+    @bp.route("/verify-rel-me/<int:field_index>", methods=["POST"])
+    @authentication_required
+    def verify_rel_me(field_index: int) -> Response:
+        user_id = session.get("user_id")
+        if not user_id:
+            return redirect(url_for("login"))
+
+        user = db.session.get(User, user_id)
+        if not user:
+            flash("User not found.")
+            return redirect(url_for("login"))
```

Comment:
I think we could incorporate this user lookup into the `authentication_required` decorator. If the user is logged in, but the user doesn't exist, we should clear the session and redirect to login wherever authentication is required.

Reply:
Removed unused route in c114b3a